plugins/LadspaEffect/caps/basics.h

```diff
@@ -76,7 +79,7 @@
 
 #define MIN_GAIN .000001 /* -120 dB */
 
-/* smallest non-denormal 32 bit IEEE float is 1.18�10-38 */
+/* smallest non-denormal 32 bit IEEE float is 1.18*10-38 */
```

Comment:
We have a moral obligation to upstream all CAPS changes.

This becomes easier to justify and easier to maintain if we do not make any minor formatting changes to CAPS source code that do not improve compilation, such as this comment.

@Rossmaxx can you explain the reasoning behind this minor formatting change?

Reply:
The x seemed to be broken, @Veratil suggested it i think.

Reply:
See https://github.com/LMMS/lmms/pull/6758#pullrequestreview-1528603538

Something changed it originally (probably IDE), so I suggested to change it to something else if it was already touched.

Reply:
@Rossmaxx I see... 

The code uses `×` (`x00D7`) or `MULTIPLICATION SIGN`, which is a bit odd to see in source code.   Let's patch it with `x` (`x0078`) or `LATIN SMALL LETTER X`  if we're going to fix it for him, otherwise we can leave it alone.

My vote it to leave it alone, since it's technically the correct symbol, but if it renders badly in text editors, the letter `x` seems better suited.

Reply:
I pasted the multiplication style from the diff. Now it seems to be corrected (atleast by looking at the diff).

Reply:
looked again, it's broken. I'mma put the lowercase `x` there now.

Reply:
> but if it renders badly in text editors

vscode displays it fine but the github diff seems to mess up.

Reply:
Thanks, looks good.